targets/targets.json

```diff
@@ -2807,6 +2807,11 @@
             "IAR",
             "GCC_ARM"
         ],
+        "supported_c_libs": {
```

Comment:
So supported_toolchains config has to be removed ?

Reply:
It is needed. Is there a reason you ask?

Reply:
no, just feeling that there is double information: supported_toolchains and supported_c_libs

Reply:
The current build tool requires it.
I suspect future versions of the build tool will remove the need for `supported_toolchains` since that information can be obtained from `supported_c_libs`.